apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/DocumentLogsTable.tsx

```diff
@@ -260,11 +259,6 @@ export const DocumentLogsTable = forwardRef<HTMLTableElement, Props>(
                     {capitalize(documentLog.source || '-')}
                   </Text.H5>
                 </TableCell>
-                <TableCell>
-                  <Text.H5 color={cellColor}>
-                    {documentLog.customIdentifier || '-'}
```

Comment:
Why?

Reply:
it's useless 99% of the time

Reply:
I think Carlos was using it when working with agents. I don't think is useless. The only improvement is adding ellipsis and noWrap. But we can open the discussion on Slack.

Reply:
i'm not removing the info just moving it to metadata, it definitely does not need to be in the table considering 99% of users don't use this value